src/app/features/change-history/entity-last-edited-info/entity-last-edited-info.component.scss

```
@@ -0,0 +1,44 @@
+  box-sizing: border-box;
+  width: max-content;
+  min-width: 220px;
+  max-width: 280px;
+  overflow-x: hidden;
+}
+
+// The notification feature sets a global `.mat-mdc-menu-panel { min-width: 310px
+// !important }`, which leaks to every menu overlay and leaves an empty right-side
+// gap in this compact info popover. Override it for our panel only (::ng-deep so
+// it reaches the body-level overlay; higher specificity + !important to win).
+::ng-deep .last-edited-menu.mat-mdc-menu-panel {
```

Stylelint (17.12.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)